### DISCUSSION TITLE
Batch testing for sensitivity analysis

### DISCUSSION THREAD

**Discussion by ellalewi547** [RangeShifter/RangeshiftR-tutorials#49] [Q&A]
Hi again, 

I am looking to do a sensitivity analysis and I wondered whether there is a way to conduct a batch test where I can run different master parameter objects together as one run so I don't need to keep checking up on my R session to see when its complete and start the run of the next one. Is this possible to do? I haven't seen any mention in the Rangeshifter Tutorials so I assume this isn't a feature for now?

Many thanks, Ella :)

**Comment by JetteReeg** [comment]
Hi Ella,

RangeShiftR package does not provide any batch feature inside the package itself. 
But you can of course generate R scripts which take care of that (for loops or even foreach loops). 
In case of a large sensitivity analyses I advise to run those kind of simulations on an HPC (and then run the simulations in parallel using e.g. a foreach function).

Best,
Jette

**Reply by ellalewi547** [reply]
Oh brilliant thank you!
